package.json

```diff
@@ -14,8 +14,9 @@
   "dependencies": {
     "ajv": "^4.10.0",
     "apollo-client": "^1.9.3",
-    "aws-sdk": "^2.36.0",
```

Comment:
any motivation behind the version update and introducing a new package below?

Reply:
Just because I had some issues while installing with npm , it turned out later that all things worked with yarn, so I'll remove these updates :)